cmake/Embed.cmake

```diff
@@ -36,13 +36,14 @@ function(embed NAME SOURCE)
       -DTEMPLATE=${CMAKE_SOURCE_DIR}/cmake/Embed.tmpl
       -P ${CMAKE_SOURCE_DIR}/cmake/EmbedRun.cmake
     DEPENDS
-      ${CMAKE_CURRENT_BINARY_DIR}/${ARG_HEX}
+      ${ARG_HEX}
       ${CMAKE_SOURCE_DIR}/cmake/EmbedRun.cmake
       ${CMAKE_SOURCE_DIR}/cmake/Embed.tmpl
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
Is this right? Doesn't it mean ${ARG_OUTPUT} goes into src/ rather than build/?

Reply:
`-DOUPTUT=...` is still specified in absolute terms, so the working directory doesn't actually matter here. I thought I had stripped these changes though, so I'll drop this.